weather/script.js

```diff
@@ -0,0 +1,15 @@
+var buttonSearch = document.getElementById('buttonSearch');
+var inputValue;
+var config = {
+    baseUrl:"http://api.apixu.com/v1/current.json?key=2f85cd8f13da458a841180955193007&q="
+};
+
+buttonSearch.addEventListener('click', function(event){
+    inputValue = document.getElementById('inputValue').value;
```

Comment:
It's better to get the input element once in the upper scope (currently you get it on every button click):
`const inputValue = document.getElementById('inputValue');`
Inside this callback you could just get `getWeather(inputValue.value)`